jupytext/cli.py

```diff
@@ -681,6 +689,9 @@ def jupytext_single_file(nb_file, args, log):
         write(notebook, nb_dest, fmt=dest_fmt)
         if args.pre_commit:
             system("git", "add", nb_dest)
+        elif args.add_untracked and is_untracked(nb_dest):
```

Comment:
What about other untracked filetypes? Using `--sync` should handle this nicely. Doesn't pre-commit hooks really shouldn't call git add inside of themselves like this. It's against the standard pattern.

Reply:
This is really bad actually because you could add files the are in the pre-commit global or hook-wide exclude . That would be very undesirable behavior.

Reply:
Yep, will remove this in favour of just exiting non-zero if output is untracked (don't add)